src/manage_container_image_scanner.c

```diff
@@ -323,6 +358,25 @@ add_container_image_scan_preferences (GHashTable *scanner_options, task_t task)
     g_hash_table_insert (scanner_options, g_strdup ("registry_allow_insecure"),
                          get_bool_string (registry_allow_insecure));
 
+  http_scanner_parsed_scans_preferences (connector, &scan_prefs);
```

Comment:
This depends on a scanner, so it could fail. Makes me think the return should be checked (and passed on to the caller). Otherwise the scan will just keep running even when there's a problem on the scanner side.